### PR TITLE
Extend register users from CSV to also add to rhods-notebooks

### PR DESCRIPTION
**Pull request by knikolla**
Related https://github.com/nerc-project/operations/issues/366

### CODE REVIEW

**Comment by hpdempsey** [comment]
@DanNiESh please review this

**Comment by DanNiESh** [comment]
Looks good to me. I've generated a csv file for Kristi to test it out.

File: tools/register_users_from_csv.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def create_user(self, username, first_name, last_name, email):
         r = self.session.get(url)
         if r.status_code == 200:
             print(f"User {username} exists.")
-        if r.status_code == 404:
+        if r.status_code in [404, 500]:
```
Comment (knikolla) [inline tools/register_users_from_csv.py:61]: This is in reference to #21 that has been fixed but the fix hasn't rolled out yet.

**Comment by knikolla** [comment]
I have processed the list that @DanNiESh gave me and all users in the list now have the `clusterrole/edit` on the `rhods-notebooks` namespace.

**Review by DanNiESh — APPROVED** [review]
Confirmed that the script works

**Pull request merged**